pandas/tests/internals/test_internals.py

```diff
@@ -1323,10 +1323,6 @@ def test_period_can_hold_element(self, element):
         elem = element(dti)
         self.check_series_setitem(elem, pi, False)
 
-    def check_setting(self, elem, index: Index, inplace: bool):
```

Comment:
helper function which wasn't being used anymore

Reply:
Looks like it was meant to de-duplicate something but never got used.  Rip it out!